docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -14,7 +14,7 @@ Nuxt automatically scans files inside these directories to register API and serv
 
 Each file should export a default function defined with `defineEventHandler()` or `eventHandler()` (alias).
 
-The handler can directly return JSON data, a `Promise` or use `event.node.res.end()` to send a response.
+The handler can directly return JSON data, a `Promise`, a `Response`, or use `event.node.res.end()` to send a response.
```

Comment:
```suggestion
The handler can directly return JSON data, a `Promise`, or use `event.node.res.end()` to send a response.
```

Reply:
Do we not support `Response` with new Nitro?